src/styles/stars.scss

```
@@ -0,0 +1,21 @@
+.stars {
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.stars__star {
+  display: flex;
+  width: 16px;
+  height: 16px;
+  background: url("../images/star.svg") no-repeat center;
+  margin: 0 4px 0 0;
```

```suggestion
  margin-right: 4px;
```
Also, make sure that the last elements of the row don't have right margin